bindings/c/test/apitester/TesterBlobGranuleErrorsWorkload.cpp

```diff
@@ -0,0 +1,199 @@
+/*
+ * TesterBlobGranuleErrorsWorkload.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "TesterApiWorkload.h"
+#include "TesterUtil.h"
+#include <memory>
+#include <fmt/format.h>
+
+namespace FdbApiTester {
+
+// FIXME: avoid duplicating this between files!
```

Comment:
I can do that. I was avoiding it because the mako blob granules implementation in should also be changed to use this implementation (the current mako implementation was written in C before mako was rewritten in C++ and is suboptimal), so I figured if we were going to do a refactoring of this, it should include the mako code. I don't think we share any code between mako and the api tester (unless I misunderstood), so I put that off.
If you think it's still better to refactor this once here for the api tester, and do a separate refactor later for mako, we can do that as well

Reply:
Yes, in this PR, I would suggest just solve it in  simple way  to avoid copying the code. Then do the mako refactoring in a separate PR.